crates/bevy_render/src/settings.rs

```diff
@@ -100,13 +100,17 @@ impl Default for WgpuSettings {
             limits
         };
 
-        let dx12_compiler =
-            wgpu::util::dx12_shader_compiler_from_env().unwrap_or(Dx12Compiler::Dxc {
-                dxil_path: None,
-                dxc_path: None,
+        let dx12_shader_compiler =
+            Dx12Compiler::from_env().unwrap_or(if !cfg!(feature = "statically-linked-dxc") {
```

Comment:
```suggestion
            Dx12Compiler::from_env().unwrap_or(if cfg!(feature = "statically-linked-dxc") {
```

I believe this was inverted by mistake. At least, now it doesn't complain anymore when I run an example without specifying a backend.